token-allocation/README.md

```diff
@@ -48,31 +50,27 @@ The adapter takes the following environment variables:
 ## Output
 ```json
 {
-    "jobRunID": "1",
-    "data": {
-        "result": 12.0322909,
-        "index": [
-            {
-                "asset": "DAI",
-                "units": "10",
-                "currency": "USD",
-                "price": "0.99985095"
-            },
-            {
-                "asset": "USDC",
-                "units": "10",
-                "currency": "USD",
-                "price": "1.00310470"
-            },
-            {
-                "asset": "USDT",
-                "units": "10",
-                "currency": "USD",
-                "price": "1.00139295"
-            }
-        ]
+  "jobRunID": "1",
+  "data": {
+    "result": 37340.964000680404,
+    "wBTC": {
+      "balance": "1",
```

Comment:
Still not sure if balance is needed in the output here, since it's already provided in the input?